tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -21,10 +21,9 @@ Build:
 {% endif %}
 {% if os in ["ubuntu2204", "rhel9", "rocky9"] %}
 # Disable Lustre installation because these newer operating systems release new kernels more often. Lustre usually does not support the latest kernels
-DevSettings:
-  Cookbook:
-    ExtraChefAttributes: |
-      {"cluster": {"lustre": {"enabled": "no" }}}
+    Installation:
```

Comment:
I thought we were moving `Installation` out of `DevSettings`

Reply:
Yes, it is out of DevSettings under Build